rust-connectors/sources/postgres/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "fluvio-connect-postgres"
+version = "0.1.0"
+edition = "2021"
+
+[[bin]]
+name = "fluvio-connect-postgres"
+path = "src/bin/main.rs"
+required-features = ["connect"]
+
+[features]
+default = ["connect"]
+connect = ["tokio", "tokio-stream", "tokio-postgres", "postgres-protocol", "tracing", "tracing-subscriber", "fluvio"]
+
+[dependencies]
+tracing = { version = "0.1", optional = true }
+tracing-subscriber = { version = "0.3", features = ["env-filter"], optional = true }
+tokio = { version = "1", features = ["full"], optional = true }
```

Comment:
I updated to use async_std as executor, but keeping tokio for `features = ["macros"]` for the `pin!` macro.

Reply:
Ah so I have discovered that we cannot switch to `async_std` because the postgres library we are using is built directly on tokio, it has started panicking because there is no active tokio runtime. I am going to revert this switch to get things working again, we can revisit this conversation if needed.